react/src/views/Box/BoxView.tsx

```diff
@@ -316,12 +341,22 @@ function BTBox() {
           numberOfItems: (boxData?.numberOfItems || 0) + (boxFormValues?.numberOfItems || 0),
         },
       })
-        .then(() => {
+        .then((res) => {
+          notificationVar({
+            title: `Box ${boxData.labelIdentifier}`,
+            type: res.errors ? "error" : "success",
+            message: res.errors
+              ? "Error while trying to change number of items in the box"
```

Comment:
"Error: Could not add/remove items from the box." should be the error. Then what happens? Does the whole action fail? Should the user try again?

Reply:
@aerinsol also here the same, the whole action fail and then user should try again